pkg/cmd/config/init/config_init.go

```diff
@@ -241,6 +254,15 @@ func cloneRepositories(
 	}, nil
 }
 
+func addHint(err error) error {
+	switch {
+	case errors.As(err, &git.RepositoryCloneErr{}):
+		return fmt.Errorf("%w: initialised already? run `corectl config update` to update repositories", err)
```

Comment:
I think it also makes sense to suggest deleting the config folder and show the path in case users want to initialize corectl from the start

Reply:
done